app/views/arf_reports/expire_reports.html.erb

```diff
@@ -0,0 +1,23 @@
+<% title _("Expire Compliance reports") %>
+
+<%= form_tag submit_expire_reports_arf_reports_path, :class => "form-horizontal well" do %>
+
+  <%= alert :class => 'alert-info', :header => _('Note'), :text => _('You are about to expire Compliance reports. All reports older than given number of day will be deleted. This action is irreversible') %>
+
+  <div class="clearfix">
+    <div class="form-group">
+      <%= label_tag("data[days]", _('Days'), :class => "col-md-2 control-label") %>
+      <div class="col-md-4">
+        <%= text_field(:data, :days, :placeholder => (_('Expire older than...')), :class => "form-control") %>
```

Comment:
`text_f` and other our helpers are meant to be used inside block supplied to `form_for` helper. I use `form_tag` since `form_for` has to be bound to a model and I just need to get a numeric value.